src/Config/AuthorizationType.cs

```diff
@@ -1,15 +1,11 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-namespace Azure.DataApiBuilder.Config
+namespace Azure.DataApiBuilder.Config;
+
```

Comment:
Lets keep the summary

Reply:
Of the Enum? What would the summary add that isn't obvious from the Enum?

Reply:
makes sense as -is. Can add back if there are more specific details that are needed which go beyond the self-explanatory enum name.